osf/models/node.py

```diff
@@ -1869,13 +1870,17 @@ def fork_node(self, auth, title=None):
         # Need to save here in order to access m2m fields
         forked.save()
 
+        if parent:
+            node_relation = NodeRelation.objects.get(parent=parent.forked_from, child=original)
+            NodeRelation.objects.get_or_create(_order=node_relation._order, parent=parent, child=forked)
+
         forked.tags.add(*self.all_tags.values_list('pk', flat=True))
         for node_relation in original.node_relations.filter(child__is_deleted=False):
             node_contained = node_relation.child
             # Fork child nodes
             if not node_relation.is_node_link:
                 try:  # Catch the potential PermissionsError above
-                    forked_node = node_contained.fork_node(auth=auth, title='')
+                    forked_node = node_contained.fork_node(auth=auth, title='', parent=forked)
                 except PermissionsError:
                     pass  # If this exception is thrown omit the node from the result set
                     forked_node = None
```

Comment:
There's a good chance that this line to the next `else` is superfluous, but didn't want to risk it in a hotfix, as it's not hurting anything (other than performance by perhaps a couple milliseconds).